jobsdb/jobsdb.go

```diff
@@ -2826,7 +2826,7 @@ func (jd *HandleT) copyJobStatusDS(tx *sql.Tx, ds dataSetT, statusList []*JobSta
 	// amount of rows are being copied in the table in a very short time and
 	// AUTOVACUUM might not have a chance to do its work before we start querying
 	// this table
-	_, err = tx.Exec(fmt.Sprintf("ANALYZE %s", ds.JobStatusTable))
+	_, err = tx.Exec(fmt.Sprintf(`ANALYZE "%s"`, ds.JobStatusTable))
```

Comment:
btw it is also possible to do this, but I am not suggesting it is better.
```suggestion
	_, err = tx.Exec(fmt.Sprintf(`ANALYZE %q`, ds.JobStatusTable))
```